### PR TITLE
Update READE.md file

### PR DESCRIPTION
**Pull request by redlands**
Updated README file.

### CODE REVIEW

**Comment by redlands** [comment]
@YuanZeng does this look good?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+We will use this project to vote for the best national parks, and recommend those parts to other people. 
+
+For example: 
+
+Name:                    # of Votes 
+Yosemite                     109
+Yellow Stone                 248
+Joshua Tree                  246
+Grand Canyon                 563
+
```
Comment (YuanZeng) [inline README.md:13]: This looks great! I will incorporate this in my National Park List file :+1:

Comment (redlands) [reply README.md:13]: Awesome, I am going to merge then. :smile:

**Comment by redlands** [comment]
close #1

**Pull request merged**